apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx

```diff
@@ -446,9 +446,9 @@ export const useTasks = ({
     alternativeNames: sanction?.entity?.otherNames,
     places: sanction?.entity?.places,
   }));
-  const ubos = pluginsOutput?.ubos?.data?.map(ubo => ({
+  const ubos = pluginsOutput?.ubo?.data?.data?.uboGraph?.map(ubo => ({
     name: ubo?.name,
-    percentage: ubo?.percentage,
+    percentage: ubo?.shareHolders?.[0]?.sharePercentage,
```

Comment:
0 index? seems like a bug

Reply:
@alonp99 this is because this is how the AsiaVerify data is structured